### PR TITLE
Connection testing

### PR DESCRIPTION
**Pull request by MantasGG**
Fixed database issues.

application.properties changes:
Changed spring.datasource.driver-class-name=org.postgresql.Driver to spring.datasource.driver-class-name=org.postgresql.Driver
Changed spring.jpa.database-platform=org.hibernate.dialect.H2Dialect to spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect

Added to databese url at the end "?sslmode=require"

Deleted:
spring.h2.console.enabled=true
spring.h2.console.path=/h2-console
spring.h2.console.settings.trace=false
spring.h2.console.settings.web-allow-others=false

gradle.build changes:
Changed runtimeOnly 'com.h2database:h2' to runtimeOnly 'org.postgresql:postgresql:42.3.3'

### CODE REVIEW

**Review by bandysiu — APPROVED** [review]
Amazing

File: src/main/java/com/team4/deskbookingappbe/model/domain/User.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+@Data
+@Entity(name = "USERS")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long USER_ID;
```
Comment (grkarolis) [inline src/main/java/com/team4/deskbookingappbe/model/domain/User.java]: Violates naming convention. should be id

File: src/main/java/com/team4/deskbookingappbe/controller/WelcomeController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.team4.deskbookingappbe.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class WelcomeController {
```
Comment (grkarolis) [inline src/main/java/com/team4/deskbookingappbe/controller/WelcomeController.java]: Why do you need this?

Comment (bandysiu) [reply src/main/java/com/team4/deskbookingappbe/controller/WelcomeController.java]: Šiukšlė, ištrinsim, buvo testui reikalinga tada

Comment (KarolisKriauciunas) [reply src/main/java/com/team4/deskbookingappbe/controller/WelcomeController.java]: For testing :)

File: src/main/java/com/team4/deskbookingappbe/model/api/CreateUserRequest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.Data;
+import org.springframework.lang.NonNull;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Data
+@AllArgsConstructor
+public class CreateUserRequest {
+    //what json keys im expecting
```
Comment (grkarolis) [inline src/main/java/com/team4/deskbookingappbe/model/api/CreateUserRequest.java]: umm, doesn't seem relevant

**Review by grkarolis — APPROVED** [review]
looks good now

**Review by KarolisKriauciunas — APPROVED** [review]

**Pull request merged**